src/qmapshack/gis/CGisWorkspace.cpp

```diff
@@ -51,6 +52,13 @@
 #include <QtWidgets>
 #include <QtXml>
 
+enum tags_hidden_e
+{
+    eTagsHiddenTrue,
+    eTagsHiddenFalse,
+    eTagsHiddenUnknown
+};
+
```

Comment:
Why is this outside of the class? Any good reason?

Reply:
I can't remeber why I didn't palce it into CGisWorkspace, that I decided not to place it into CGisItemRate because it has nothing to do with that.
So probably the answer is no.

Reply:
Then place it into the class.